website/docs/r/secure_managed_policy.md

```diff
@@ -46,7 +46,8 @@ resource "sysdig_secure_managed_policy" "sysdig_runtime_threat_detection" {
 
 * `name` - (Required) The name of the Secure managed policy. It must match the name of an existing managed policy.
 
-* `type` - (Optional) Specifies the type of the runtime policy. Must be one of: `falco`, `list_matching`, `k8s_audit`, `aws_cloudtrail`. By default it is `falco`.
+* `type` - (Optional) Specifies the type of the runtime policy. Must be one of: `falco`, `list_matching`, `k8s_audit`,
```

Comment:
do we have gcp/azure as supported policy types?

Reply:
Yes, we do.  I checked in the API and the validation that terraform performs also allows gcp/azure.  We just never updated the documentation.